src/Psl/Str/convert_encoding.php

```diff
@@ -0,0 +1,19 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Psl\Str;
+
+use function mb_convert_encoding;
+
+/**
+ * Detect the encoding of the giving string.
+ *
+ * @psalm-return null|string The string encoding or null if unable to detect encoding.
```

Comment:
Description seems to be incorrect.

see: https://www.php.net/manual/en/function.mb-convert-encoding.php